assets/js/components/UserMenu/index.js

```diff
@@ -162,6 +162,35 @@ export default function UserMenu() {
 		return null;
 	}
 
+	let accountLabel;
+
+	if ( userFullName && userEmail ) {
+		accountLabel = sprintf(
+			/* translators: Account info text. 1: User's (full) name 2: User's email address. */
+			__( 'Google Account for %1$s (Email: %2$s)', 'google-site-kit' ),
+			userFullName,
+			userEmail
+		);
+	}
+
+	if ( userFullName && ! userEmail ) {
+		accountLabel = sprintf(
+			/* translators: Account info text. 1: User's (full) name. */
+			__( 'Google Account for %1$s', 'google-site-kit' ),
+			userFullName,
+			userEmail
+		);
+	}
+
+	if ( ! userFullName && userEmail ) {
+		accountLabel = sprintf(
+			/* translators: Account info text. 1: User's email address. */
+			__( 'Google Account (Email: %1$s', 'google-site-kit' ),
+			userFullName,
```

Comment:
Same here, `userFullName` isn't needed right? This actually makes this translation incorrect.

Reply:
Argh, thanks 😅